src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = days * 40;
+
+  if (days >= 3 && days < 7) {
+    cost -= 20;
+  }
+
+  if (days >= 7) {
+    cost -= 50;
+  }
```

Comment:
```suggestion
  if (days >= 7) {
    cost -= 50;
  } else if (days >= 3) {
    cost -= 20;
  }
```
Change these `if`s by places and you will get rid of one redundant check